notebooks/lesson04_intro-to-random-forests.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>or it could be houses that are "special" due to reasons that are not reflected in the data (e.g. a historic building, special architect, damages etc)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>